src/Compilers/Test/Resources/Core/TestResources.cs

```diff
@@ -78,6 +78,21 @@ public static class ExpressionCompiler
         private static byte[] s_windowsStorage;
         public static byte[] WindowsStorage => ResourceLoader.GetOrCreateResource(ref s_windowsStorage, "ExpressionCompiler.Windows.Storage.winmd");
     }
+
+    public static class GeneratorTests
+    {
+        private static byte[] s_netCoreGenerator;
+        public static byte[] NetCoreGenerator => ResourceLoader.GetOrCreateResource(ref s_netCoreGenerator, "Generators.NetCoreGenerator.dll");
+
+        private static byte[] s_netFrameworkGenerator;
+        public static byte[] NetFrameworkGenerator => ResourceLoader.GetOrCreateResource(ref s_netFrameworkGenerator, "Generators.NetFrameworkGenerator.dll");
+
+        private static byte[] s_netStandardGenerator;
+        public static byte[] NetStandardGenerator => ResourceLoader.GetOrCreateResource(ref s_netStandardGenerator, "Generators.NetStandardGenerator.dll");
+
+        private static byte[] s_noTargetGenerator;
+        public static byte[] NoTargetGenerator => ResourceLoader.GetOrCreateResource(ref s_noTargetGenerator, "Generators.NoTargetGenerator.dll");
```

Comment:
Rather than checking in DLLs, which continue to inflate our repo size, why not just build them as a part of the tests that need them?

Reply:
Ah yeah didn't think of that, plus now we're not actually dealing with references this is much simpler to do.